.gitignore

```diff
@@ -12,6 +12,7 @@ staging/
 /typings/
 !staging/.gitkeep
 /semantic/
+src/**/*.css
```

Comment:
We don't want compile css in the repo, they are generated when we create an snapshot and we had to clean them up manually, so not they are ignored.